cms/djangoapps/export_course_metadata/tasks.py

```diff
@@ -27,5 +27,5 @@ def export_course_metadata_task(self, course_key_string):  # pylint: disable=unu
     """
     course_key = CourseKey.from_string(course_key_string)
     highlights = get_all_course_highlights(course_key)
-    highlights_content = ContentFile(json.dumps({'highlights': highlights}))
+    highlights_content = ContentFile(json.dumps({'highlights': highlights}).encode('utf-8'))
```

Comment:
Huh, this is apparently an open bug in django-storages with a PR to fix it that's been sitting around unmerged for almost 3 years for no particularly good reason: https://github.com/jschneier/django-storages/pull/911